.github/workflows/net-perf-gke.yaml

```diff
@@ -157,42 +157,29 @@ jobs:
       - name: Set Environment Variables
         uses: ./.github/actions/set-env-variables
 
+      - name: Get Cilium's default values
+        id: default_vars
+        uses: ./.github/actions/helm-default
+        with:
+          image-tag: ${{ inputs.SHA || github.sha }}
+          chart-dir: ./untrusted/install/kubernetes/cilium
+
       - name: Set up job variables
         id: vars
         run: |
           if [ "${{ github.event_name }}" = "workflow_dispatch" ] || [ ${{ github.event.pull_request }} ] ; then
-            SHA="${{ inputs.SHA }}"
             OWNER="${{ inputs.PR-number }}"
+            SHA="${{ inputs.SHA }}"
```

Comment:
SHA looks unused, and could be dropped.

Reply:
SHA is appended to `$GITHUB_OUTPUT` at line 194. This was in the original code as well (line 179), so I assume it is needed?

Reply:
It seems you actually removed the only usage, because I cannot find any occurrence of `steps.vars.outputs.sha`.

Reply:
Ah good, then I'll remove it.